testsuite/regressiontests/issue-311.sli

```diff
@@ -0,0 +1,75 @@
+/*
+ *  issue-311.sli
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/* BeginDocumentation
+
+Name: testsuite::issue-311 Ensure NEST behaves well after exception during update
+
+Synopsis: (issue-311) run -> NEST exits if test fails
+
+Description:
+This test ensures that NEST behaves properly when a model triggers and
+exception during update.
+
+Author: Hans Ekkehard Plesser, 2016-11-02
+ */
```

Comment:
We could even add: https://github.com/nest/nest-simulator/issues/311 to help people find the original issue...